### ISSUE TITLE
sudo modprobe rtl8723be ant_sel=2 modprobe: ERROR: could not insert 'rtl8723be': Unknown symbol in module, or unknown parameter (see dmesg)

### ISSUE THREAD

**Issue by andrew19771211** [lwfinger/rtlwifi_new#538]
sudo modprobe rtl8723be ant_sel=2
modprobe: ERROR: could not insert 'rtl8723be': Unknown symbol in module, or unknown parameter (see dmesg)
my system is:
linux-headers-5.3.0-40-generic ubuntu 19.10

and@and-HP-250-G4-Notebook-PC:~/rtlwifi_new$ lspci -v | grep -iA 7 network
0d:00.0 Network controller: Realtek Semiconductor Co., Ltd. RTL8723BE PCIe Wireless Network Adapter
	Subsystem: Hewlett-Packard Company RTL8723BE PCIe Wireless Network Adapter
	Flags: fast devsel, IRQ 7
	I/O ports at 3000 [size=256]
	Memory at c4000000 (64-bit, non-prefetchable) [size=16K]
	Capabilities: <access denied>
	Kernel modules: rtl8723be
lspci -nn | grep -i net

07:00.0 Ethernet controller [0200]: Realtek Semiconductor Co., Ltd. RTL8111/8168/8411 PCI Express Gigabit Ethernet Controller [10ec:8168] (rev 15)
0d:00.0 Network controller [0280]: Realtek Semiconductor Co., Ltd. RTL8723BE PCIe Wireless Network Adapter [10ec:b723]

**Comment by lwfinger** [comment]
The title of this post has the statement that you should "See DMESG"!! Where is that information?

The most common cause of this error is that your kernel source does not match your running kernel. When you do the "make" statement, the first line of output will be something like

make -C /lib/modules/5.6.0-rc4-00351-gbb32833b4413/build M=/home/finger/rtlwifi_new1 modules

The part of the line following "modules" must match the output of the "uname -r" command. On my system, that is 

finger@linux-n6zm:~/rtlwifi_new1> uname -r
5.6.0-rc4-00351-gbb32833b4413

Thus my kernel matches my build. Yours must as well. Please post the appropriate dmesg output if your system passes the above test.

**Comment by andrew19771211** [comment]
Hi
Here my outputs

and@and-HP-250-G4-Notebook-PC:~/rtlwifi_new$ make
make -C /lib/modules/5.3.0-40-generic/build M=/home/and/rtlwifi_new modules

and@and-HP-250-G4-Notebook-PC:~/rtlwifi_new$ uname -r
5.3.0-40-generic


and@and-HP-250-G4-Notebook-PC:~$ dmesg | grep rtlwifi
[    4.205460] rtlwifi: loading out-of-tree module taints kernel.
[    4.209333] rtlwifi: module verification failed: signature and/or required key missing - tainting kernel

Thanks

**Comment by lwfinger** [comment]
Turn off secure boot in your BIOS. If you do not do that, then you will need to learn how to sign your modules.

**Comment by Cedarpcf** [comment]
I have the same issue. Been trying for days to resolve it. I haven't found any valid solution even after reading several articles and watching several videos that claim to solve it.
I am new to Linux, I don't know these technical stuffs and I really need help on what to do. Thank you.

BELOW IS THE ERROR I GET:
cedarpcf@Cedar-PC:~/rtlwifi_new$ make
make -C /lib/modules/5.0.0-32-generic/build M=/home/cedarpcf/rtlwifi_new modules
make[1]: Entering directory '/usr/src/linux-headers-5.0.0-32-generic'
  CC [M]  /home/cedarpcf/rtlwifi_new/btcoexist/halbtc8723b2ant.o
In file included from /home/cedarpcf/rtlwifi_new/btcoexist/halbt_precomp.h:25:0,
                 from /home/cedarpcf/rtlwifi_new/btcoexist/halbtc8723b2ant.c:14:
/home/cedarpcf/rtlwifi_new/btcoexist/../wifi.h:43:2: error: #error "This branch is abandoned. Please do not use. As an alternative, 'git checkout origin/extended -b extended'"
 #error "This branch is abandoned. Please do not use. As an alternative, 'git checkout origin/extended -b extended'"
  ^~~~~
scripts/Makefile.build:284: recipe for target '/home/cedarpcf/rtlwifi_new/btcoexist/halbtc8723b2ant.o' failed
make[3]: *** [/home/cedarpcf/rtlwifi_new/btcoexist/halbtc8723b2ant.o] Error 1
scripts/Makefile.build:515: recipe for target '/home/cedarpcf/rtlwifi_new/btcoexist' failed
make[2]: *** [/home/cedarpcf/rtlwifi_new/btcoexist] Error 2
Makefile:1614: recipe for target '_module_/home/cedarpcf/rtlwifi_new' failed
make[1]: *** [_module_/home/cedarpcf/rtlwifi_new] Error 2
make[1]: Leaving directory '/usr/src/linux-headers-5.0.0-32-generic'
Makefile:57: recipe for target 'all' failed
make: *** [all] Error 2
cedarpcf@Cedar-PC:~/rtlwifi_new$

**Comment by Cedarpcf** [comment]
I have the same issue. Been trying for days to resolve it. I haven't found any valid solution even after reading several articles and watching several videos that claim to solve it.
I am new to Linux, I don't know these technical stuffs and I really need help on what to do. Thank you.

BELOW IS THE ERROR I GET:

![image](https://user-images.githubusercontent.com/66806470/84449596-c8372700-ac45-11ea-9b30-9ae0732632cb.png)

**Comment by lwfinger** [comment]
I got tired of all the stupid posts from people that do not read anything, including the error messages. That is the reason I abandoned this repository. I kept the rtw88 branch as that has drivers that are not found in current kernels.

Driver rtl8723be has been in the kernel since version 3.15. You are using kernel 5.0. DO NOT USE ANY EXTERNAL MODULE for RTL8723BE!!!!! Use your kernel's version.

Is that not clear?
